src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonDrawer/PolygonDrawer.tsx

```diff
@@ -201,11 +201,7 @@ const PolygonDrawer = ({
   }, [isOpenPolygonDrawer]);
 
   useEffect(() => {
-    if (
-      selectedPolygonData &&
-      Object.keys(selectedPolygonData as SitePolygon).length === 0 &&
-      isEmpty(polygonSelected)
-    ) {
+    if (selectedPolygonData && isEmpty(selectedPolygonData as SitePolygon) && isEmpty(polygonSelected)) {
```

Comment:
🎨 I'll approve as-is, but I don't think the `as SitePolygon` is doing anything especially useful here. `isEmpty()` doesn't care about the type being passed in.